tests/ui/job-view/SecondaryNavBar_test.jsx

```diff
@@ -76,4 +82,24 @@ describe('SecondaryNavBar', () => {
     expect(await waitForElement(() => getByText('22'))).toBeInTheDocument();
     expect(await waitForElement(() => getByText('10'))).toBeInTheDocument();
   });
+
+  test('should call updateButtonClick, on revision changed button click', async () => {
```

Comment:
Nicely done on this test!  :)

Reply:
Tests are good :) 
Tests are good :) 

> Please finish the second part of the assignment and/or finalize your application.

Are you referring to additional test on `app_test.jsx`? or something bug related? can't see anything else in the bug report?

Reply:
I just meant please finalize your application, sorry.  Poor wording on my part.  :)